internal/store/store.go

```diff
@@ -31,8 +32,16 @@ import (
 var userAgent = fmt.Sprintf("%s/%s (%s;%s)", config.ToolName, version.Version, runtime.GOOS, runtime.GOARCH)
 
 const (
-	atlasAPIPath = "api/atlas/v1.0/"
-	yes          = "yes"
+	atlasAPIPath          = "api/atlas/v1.0/"
+	yes                   = "yes"
+	responseHeaderTimeout = 10 * time.Minute
+	tlsHandshakeTimeout   = 10 * time.Second
+	timeout               = 10 * time.Second
+	keepAlive             = 30 * time.Second
+	maxIdleConns          = 10
```

Comment:
Is this meant to be `100`? If not, why the drastic change?

Reply:
originally I got this from some recommendations for web server, but the cli should not use more than 3 or 4 connections given it's "one off" nature